canonical/charmhub.io
# Fix charms architecture not showing

## Done
- Fix charms architecture dropdown not showing all architectures
- Fix charms bases not showing all
## How to QA
- Go to https://charmhub-io-1467.demos.haus/autocert-charmers-autocert
- Click on the charms channel button, click the `architecture` dropdown and ensure the architectures are displayed
- Also, check the runs on column and ensure all bases for a particular channels are shown
## Issue / Card WD-760 
## Issue / Card WD-761
Fixes #1416 

## Screenshots
<img width="215" alt="image" src="https://user-images.githubusercontent.com/47438585/213707856-32c5a880-c4d1-4947-834b-c9298929b2b2.png">

<img width="638" alt="image" src="https://user-images.githubusercontent.com/47438585/213710739-c7d64806-613a-4d5b-b519-d6ea2e790be0.png">
